content/issues/2024-02-26-fedora-auth-problem.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Fedora infrastructure authentication problem"
+date: "2024-02-26T13:00:00+00:00"
+affected:
+  - Pull from upstream
+  - Propose downstream
+  - Koji
+  - Bodhi
+resolved: false
+#resolvedWhen:
```

Comment:
```suggestion
resolved: true
resolvedWhen: "2024-02-26T14:28:00+00:00"
```

If I got it correctly they have resolved it.

Reply:
nice, will update it